code/modules/mob/living/living_status_procs.dm

```diff
@@ -112,7 +112,7 @@ STATUS EFFECTS
 /mob/living/proc/on_lying_down(new_lying_angle)
 	if(layer == initial(layer)) //to avoid things like hiding larvas.
 		if(HAS_TRAIT(src, TRAIT_CONTORTED_BODY))
-			layer = TURF_LAYER + 0.2
+			layer = BLASTDOOR_LAYER
```

Comment:
Im pretty sure this means they won't be able to go under firelocks. Not sure if you want that or not.